app/uk/gov/hmrc/economiccrimelevyregistration/models/BusinessSector.scala

```diff
@@ -83,4 +93,20 @@ object BusinessSector {
 
     override def writes(o: BusinessSector): JsValue = JsString(o.toString)
   }
+
+  def transformFromSubscriptionResponse(businessSector: String): BusinessSector =
+    businessSector.replace(" ", "").toLowerCase() match {
```

Comment:
you can use `businessSector.filterNot(_.isWhitespace).toLowerCase`

Reply:
Done